### PR TITLE
initial draft

### PR DESCRIPTION
**Pull request by pmwtsn**

### CODE REVIEW

File: guide/src/install.md [hunk 1]
```diff
@@ -1,5 +1,114 @@
...
+
+    $ source ~/.profile
+
+Check `JAVA_HOME` by running:
+
+    $ echo $JAVA_HOME
+
+You should see something like:
+
+    java version “1.7.0_45"
```
Comment (charleso) [inline guide/src/install.md]: @pmwtsn It's probably not an issue, but maybe watch/avoid those "smart quotes".

File: guide/src/install.md [hunk 2]
```diff
@@ -1,5 +1,114 @@
...
+    $ source ~/.profile
+
+Now run Ivory:
+
+    $ ivory —-help
+
+You should see:
+
+    Ivory 1.0.0-cdh5-20141113112607-b49d4ba
+    Usage: {rename|cat-dictionary|cat-errors|cat-facts|chord|config|convert-dictionary|count-facts|create-repository|debug-dump-facts|debug-dump-reduction|fact-diff|import-dictionary|ingest|recompress|recreate|snapshot|factset-statistics}
```
Comment (charleso) [inline guide/src/install.md]: FYI This will look (quite) different in the latest version

File: guide/src/install.md [hunk 3]
```diff
@@ -1,5 +1,114 @@
...
+
+You should see something like:
+
+    /usr/local/hadoop
+
+## Ivory
+
+Install Ivory by rumming:
+
+    $ curl -OfsSL https://raw.githubusercontent.com/ambiata/ivory/master/bin/install
```
Comment (blever) [inline guide/src/install.md]: Note that whilst this script is correct, what it pulls down is a very out of date version of Ivory (unless somebody fixed that). /cc @charleso

**Comment by blever** [comment]
@pmwtsn by the way - welcome!

**Comment by markhibberd** [comment]
Closed via #654

**Pull request closed without merging**